apps/admin-x-activitypub/src/views/Explore/Explore.tsx

```
@@ -0,0 +1,142 @@
+                            type='primary'
+                            onFollow={onFollow}
+                            onUnfollow={onUnfollow}
+                        /> :
+                        <div className='inline-flex items-center'>
+                            <Skeleton className='w-12' />
+                        </div>
+                    }
+                </div>
+                {profile.actor.summary &&
+                    <div
+                        dangerouslySetInnerHTML={{__html: profile.actor.summary}}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)